proxy/secret.go

```diff
@@ -0,0 +1,60 @@
+package proxy
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	"github.com/openfaas/faas-cli/schema"
+)
+
+//GetSecretList get secrets list
+func GetSecretList(gateway string, tlsInsecure bool) ([]schema.Secret, error) {
+	var results []schema.Secret
```

Comment:
We probably want a "warn insecure" block here like we do in the deploy code https://github.com/openfaas/faas-cli/blob/200359ce2237dd8b55e156683957c44ab26b2ef9/proxy/deploy.go#L81-L85

Reply:
+1 @viveksyngh